modules/core/src/main/kotlin/com/metalbear/mirrord/MirrordExecManager.kt

```diff
@@ -123,7 +123,21 @@ class MirrordExecManager(private val service: MirrordProjectService) {
 
         val mirrordApi = service.mirrordApi(projectEnvVars)
 
-        val mirrordConfigPath = projectEnvVars?.get(CONFIG_ENV_NAME)
+        val mirrordConfigPath = projectEnvVars?.get(CONFIG_ENV_NAME)?.let {
+            if (it.contains("\$ProjectPath\$")) {
+                service.project.basePath?.let { projectPath ->
```

Comment:
Docs for `Project::getBasePath` (auto translated to `basePath` property here) say `It's <b>strongly recommended</b> to use other methods instead of this one`.

I remember having multiple problems when working with default mirrord configs (which required finding the project path). You can see the final result in `MirrordConfigAPI::getMirrordDirParent` method.